### PR TITLE
Update open source community files

### PR DESCRIPTION
**Pull request by Jonasvdbo**
Updated the readme.md and added the following open source community files:

- CONTRIBUTING.md
- CODE_OF_CONDUCT.md
- PROJECT_GOVERNANCE.md
- SUPPORT.md

Signed-off-by: Jonas van den Bogaard 3628277+Jonasvdbo@users.noreply.github.com

### CODE REVIEW

**Comment by Jonasvdbo** [comment]
@TonyXiang8787 Can you review this PR?

**Review by TonyXiang8787 — CHANGES_REQUESTED** [review]
also add requirements file

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+SPDX-License-Identifier: MPL-2.0
+-->
+# Power Gird Model Benchmark
+`power-grid-model-benchmark` is a benchmark example of [power-grid-model](https://github.com/alliander-opensource/power-grid-model) in a jupyter notebook on your local machine.
+
+## Instructions
+
+### Locally
+Install:
+`pip install -r requirements.txt`
```
[the remark this answers is not in the document]
Comment (Jonasvdbo) [reply README.md:13]: Not fully understand what you mean @TonyXiang8787. My understanding is that there is a requirements file: https://github.com/Jonasvdbo/power-grid-model-benchmark/blob/Update-open-source-community-files/requirements.txt

[the remark this answers is not in the document]
Comment (TonyXiang8787) [reply README.md:13]: You are correct. Please add license header also to `requirements.txt`.

Moreover, you can just uncomment `power-grid-model` there without any index url. It is now an public PyPI package.

**Review by TonyXiang8787 — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jonasvdbo) [reply README.md:13]: Good remarks. I will update the `requirements.txt`.

**Pull request merged**